docs/conf.py

```diff
@@ -4,17 +4,40 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+from functools import partial
+from pathlib import Path
+
+from setuptools_scm import get_version
 from sphinx.application import Sphinx
 from sphinx.util.docutils import SphinxDirective
 
-from myst_parser import __version__
+# -- Path setup --------------------------------------------------------------
 
+PROJECT_ROOT_DIR = Path(__file__).parents[1].resolve()  # pylint: disable=no-member
+get_scm_version = partial(get_version, root=PROJECT_ROOT_DIR)
 # -- Project information -----------------------------------------------------
 
+github_url = "https://github.com"
+github_repo_org = "ansible"
+github_repo_name = "ansible-language-server"
+github_repo_slug = f"{github_repo_org}/{github_repo_name}"
+github_repo_url = f"{github_url}/{github_repo_slug}"
+github_sponsors_url = f"{github_url}/sponsors"
```

Comment:
Is this a copy-pasta? 
Can this be all moved down with the extlinks configuration: I'm guessing that is only where it is being used

Reply:
It is. I have it here in all the projects because it's where the common args are defined + these can be reused in other places if necessary. Keeping such things in the same place across projects, makes sure that when I sync the configuration into various repos, there's not many conflicts, it's easier from the maintenance perspective.